### DISCUSSION TITLE
How to get UVR to use free CPU/RAM resources on MacOS?

### DISCUSSION THREAD

**Discussion by ynnckstrm** [Anjok07/ultimatevocalremovergui#1141] [Q&A]
Hi there,

I'm using UVR to convert a .wav into vocal + instrumental using MDX-NET using MDX23C-InstVocHQ.
One thing I ran into is that UVR is not using all system resources, even though I don't see any particular reason for it to not use them.
My CPU is at 40% and my unified RAM has some spare GBs as well.

I'm using a 2023 MacBook Pro M3, 16GB, 1TB. Unlike on windows I don't see any way to prioritize UVR's process in the activity monitor. Can anyone explain the matter, or help me change it?

<img width="957" alt="Bildschirmfoto 2024-01-31 um 12 40 08" src="https://github.com/Anjok07/ultimatevocalremovergui/assets/19638454/30e66435-6881-4a9a-bf30-5819d40701cc">
<img width="961" alt="Bildschirmfoto 2024-01-31 um 12 40 16" src="https://github.com/Anjok07/ultimatevocalremovergui/assets/19638454/ce957d88-2537-4951-8c10-012cb88d87c8">

Thank you very much! ☺️

**Comment by Saiborg23** [comment]
Just wanted to bump this thread. I'm looking to fully utilize the CPU of my M1 max.
